DocumentationAnalyzers/DocumentationAnalyzers/PortabilityRules/DOC207UseSeeLangwordCorrectly.cs

```diff
@@ -0,0 +1,100 @@
+        private static readonly LocalizableString Description = new LocalizableResourceString(nameof(PortabilityResources.DOC207Description), PortabilityResources.ResourceManager, typeof(PortabilityResources));
+
+        private static readonly DiagnosticDescriptor Descriptor =
+            new DiagnosticDescriptor(DiagnosticId, Title, MessageFormat, AnalyzerCategory.PortabilityRules, DiagnosticSeverity.Warning, AnalyzerConstants.EnabledByDefault, Description, HelpLink);
+
+        /// <inheritdoc/>
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; }
+            = ImmutableArray.Create(Descriptor);
+
+        public override void Initialize(AnalysisContext context)
+        {
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
+            context.EnableConcurrentExecution();
+
+            context.RegisterSyntaxNodeAction(HandleXmlNodeSyntax, SyntaxKind.XmlElement, SyntaxKind.XmlEmptyElement);
+        }
+
+        private static void HandleXmlNodeSyntax(SyntaxNodeAnalysisContext context)
+        {
+            var xmlNodeSyntax = (XmlNodeSyntax)context.Node;
+            var name = xmlNodeSyntax.GetName();
+            if (name is null || name.Prefix != null)
+            {
+                return;
+            }
+
+            if (name.LocalName.ValueText != XmlCommentHelper.SeeXmlTag)
+            {
+                return;
+            }
+
+            SyntaxList<XmlAttributeSyntax> attributes;
+            if (xmlNodeSyntax is XmlEmptyElementSyntax xmlEmptyElement)
+            {
+                attributes = xmlEmptyElement.Attributes;
+            }
+            else
+            {
+                attributes = ((XmlElementSyntax)xmlNodeSyntax).StartTag.Attributes;
+            }
+
+            foreach (var attribute in attributes)
+            {
+                if (attribute.Name is null || attribute.Name.Prefix != null)
+                {
+                    continue;
+                }
+
+                if (attribute.Name.LocalName.ValueText != XmlCommentHelper.LangwordArgumentName)
+                {
+                    continue;
+                }
+
+                var text = ((XmlTextAttributeSyntax)attribute).TextTokens;
+                string valueText;
+                if (text.Count == 1)
+                {
+                    valueText = text[0].ValueText;
+                }
+                else
+                {
+                    valueText = string.Join(string.Empty, text.Select(textToken => textToken.ValueText));
+                }
+
+                if (SyntaxFacts.GetKeywordKind(valueText) != SyntaxKind.None
+                    || SyntaxFacts.GetContextualKeywordKind(valueText) != SyntaxKind.None)
+                {
+                    continue;
+                }
+
+                context.ReportDiagnostic(Diagnostic.Create(Descriptor, attribute.Name.LocalName.GetLocation()));
```

Comment:
I wonder if the diagnostic should appear on the attribute value rather than the name. An error message of "langword value must be a keyword" or similar would make the problem/fix clearer to the user.

Reply:
> I wonder if the diagnostic should appear on the attribute value rather than the name.

I figured this addressed the confusion that could arise if an element had multiple attributes.

> An error message of "langword value must be a keyword" or similar would make the problem/fix clearer to the user.

👍